tasks/client_config.yaml

```
@@ -1,10 +1,33 @@
 ---
+- name: Check for root password
+  changed_when: false
+  stat:
+    path: /usr/local/etc/mysql-password
+  register: mariadb_mysql_password
+
+- name: create root password
+  when: mariadb_mysql_password.stat.exists == false
+  copy:
+    dest: /usr/local/etc/mysql-password
+    content: "{{ lookup('password', '/tmp/passwordfile length=45 chars=ascii_letters,digits') }}"
```

Since we don't use that file can we write to `/dev/null` instead of `/tmp/passwordfile`? Also update length to match the proServer standard.
```suggestion
    content: "{{ lookup('password', '/dev/null length=32 chars=ascii_letters,digits') }}"
```

---

I have seen no standard document :-P and inside the proServer-project lengths are different, so be careful calling it a standard ;-)